### PR TITLE
Remove emoji_regex dependency

### PR DESCRIPTION
**Pull request by ticky**
I'd been looking into releasing an update to this gem of mine, and I was going to smoke test it by upgrading it here, but it turns out the one thing we still used it for (since Buildkite doesn't regex search for Unicode emoji, only by name) was to work around an issue with our data source, which appears to have since been resolved by them

### CODE REVIEW

File: all_emoji_names.txt [hunk 1]
```diff
@@ -215,7 +215,6 @@ batman
 battery
 bazel
 beach_with_umbrella
-beans
```
Comment (waferbaby) [inline all_emoji_names.txt:218]: Were these leaving intentional?

**Review by waferbaby — APPROVED** [review]
Looks good - only unsure about the emoji names being removed.

File: all_emoji_names.txt (hunk 1 above)
Comment (ticky) [reply all_emoji_names.txt:218]: I believe the names were changed upstream as of #369, but it seems like some of the changes were missed?

Comment (yob) [reply all_emoji_names.txt:218]: weird - the lines removed in this PR were **added** in #369.

I only have a loose understanding of what `all_emoji_names.txt` is for, but why would they be added and then removed?

**Comment by ticky** [comment]
There! Much better :)

**Review by yob — APPROVED** [review]
down with dependencies! 👇👇

**Comment by ticky** [comment]
Take that, my second ever Ruby Gem, now you're only depended on by [checks notes] fastlane??? oh no

**Pull request merged**